cmd/katapult/data_centers.go

```diff
@@ -46,23 +49,17 @@ func getDataCenterCmd(client dataCentersClient) *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 		Short: "Get details for a data center",
 		Long:  "Get details for a data center.",
-		RunE: func(cmd *cobra.Command, args []string) error {
+		RunE: renderOption(func(cmd *cobra.Command, args []string) (Output, error) {
```

Comment:
I'm not a huge fan of the `renderOption` function name, as in this context here it doesn't feel that obvious of what it does. Admittedly a good name for this is difficult, best ones I can think of off the top of my head are `outputWrapper` and `cmdRunEFunc`.